src/System.Runtime/tests/System.Runtime.Tests.csproj

```diff
@@ -97,8 +97,10 @@
   </ItemGroup>
   <ItemGroup Condition="'$(TargetGroup)' == 'netstandard1.7'">
     <Compile Include="ArrayTests.netstandard1.7.cs" />
-    <Compile Include="EnumTests.netstandard1.7.cs" />
+    <Compile Include="CompareToTests.netstandard1.7.cs" />
```

Comment:
>CompareToTests.netstandard1.7 [](start = 22, length = 29)

If there is no CompareToTests.cs file yet (which it looks like there isn't) I would prefer if these files don't have the `.netstandard1.7` suffix. The reason is that we will eventually ship netsandard1.8 and so on, and we want these tests to apply to there as well and the name might make it a bit confusing. I'm totally fine of adding the suffix though when the file without the suffix already exists.